integration-tests/test-extension/extension/deployment/src/main/java/io/quarkus/extest/deployment/ModulesCustomProcessor.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.extest.deployment;
+
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.ModuleOpenBuildItem;
+
+public class ModulesCustomProcessor {
+
+    @BuildStep
+    ModuleOpenBuildItem openModules() {
+        return new ModuleOpenBuildItem("java.base", ModuleOpenBuildItem.ALL_UNNAMED, "java.util");
```

Comment:
What is requiring this? Is this just for a test to consume?

Reply:
Yes this is in the testing extension for integration tests to verify this all.